content/en/docs/tutorials/kubernetes-basics/_index.html

```diff
@@ -9,109 +9,175 @@
   title: Walkthrough the basics
 ---
 
-{{% katacoda-removal %}}
-
 <!DOCTYPE html>
 
 <html lang="en">
+  <body>
+    <div class="layout" id="top">
+      <main class="content">
+        <div class="row">
+          <div class="col-md-9">
+            <h2>Kubernetes Basics</h2>
+            <p>
+              This tutorial provides a walkthrough of the basics of the
+              Kubernetes cluster orchestration system. Each module contains some
+              background information on major Kubernetes features and concepts,
+              and includes an interactive online tutorial. These interactive
+              tutorials let you manage a simple cluster and its containerized
+              applications for yourself.
+            </p>
+            <p>Using the interactive tutorials, you can learn to:</p>
+            <ul>
+              <li>Deploy a containerized application on a cluster.</li>
+              <li>Scale the deployment.</li>
+              <li>
+                Update the containerized application with a new software
+                version.
+              </li>
+              <li>Debug the containerized application.</li>
+            </ul>
+            <p>
+              The tutorials use Katacoda to run a virtual terminal in your web
+              browser that runs Minikube, a small-scale local deployment of
+              Kubernetes that can run anywhere. There's no need to install any
+              software or configure anything; each interactive tutorial runs
+              directly out of your web browser itself.
```

Comment:
This text is wrong. Please either amend it, or change this PR to not touch it.

Reply:
Do you want to have other text there or shall I simply remove any mention of katacoda on the page?

Reply:
For now I have removed the text. Let me know if you want to add something good there

Reply:
Is it worthwhile to leave the shortcode? I don't know if this notice is still up on other pages.

Reply:
shortcode? Sorry I didn't get you.
Are you talking about the paragraph which mentions about the "use of katacoda" in tutorials?